scripts/helmAcs.sh

```diff
@@ -81,6 +81,10 @@ else
               REGISTRYCREDENTIALS="$2";
               shift 2
```

Comment:
in the beginning please add
if [ $# -lt 12 ]; then

Reply:
Sure. Np!